fastai/text/models/awdlstm.py

```diff
@@ -25,7 +30,12 @@ def forward(self, x):
 class WeightDropout(Module):
     "A module that wraps another layer in which some weights will be replaced by 0 during training."
 
-    def __init__(self, module, weight_p, layer_names='weight_hh_l0'):
+    def __init__(
+        self,
+        module:nn.Module, # wrapped module
+        weight_p:float, # weight dropout probability
+        layer_names:(str,Sequence[str])='weight_hh_l0' # names of the parameters to apply dropout to
+    ):
```

Comment:
Per the recent update to the [style guide](https://github.com/muellerzr/fastai-docment-sprint/blob/master/docs_src/style-guide.md), could you remove all the newlines from the self and cls args like the example below?

And also capitalize the first word of all the docment comments? Thanks.


```suggestion
    def __init__(self,
        module:nn.Module, # Wrapped module
        weight_p:float, # Weight dropout probability
        layer_names:(str,Sequence[str])='weight_hh_l0' # Names of the parameters to apply dropout to
    ):
```